agent/fw_laravel.c

```diff
@@ -959,7 +961,9 @@ NR_PHP_WRAPPER(nr_laravel_application_construct) {
 
   if (NRINI(vulnerability_management_package_detection_enabled)) {
     // Add php package to transaction
-    nr_txn_add_php_package(NRPRG(txn), "laravel/framework", version);
+    nr_txn_add_php_package(NRPRG(txn), PHP_PACKAGE_NAME, version);
+    nr_fw_support_add_package_supportability_metric(
+        NRPRG(txn), PHP_PACKAGE_NAME, version);
```

Comment:
shouldn't we add the metric even if vm is disabled?

Reply:
Good point, fixed: [647d424](https://github.com/newrelic/newrelic-php-agent/pull/868/commits/647d424ffb824368d14f803f6812df43a75be043)